ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift

```diff
@@ -90,15 +90,15 @@ class DateTimePickerDemoController: DemoController {
     }
 
     func calcDatePickerParams() -> (startDate: Date, endDate: Date, titles: DateTimePicker.Titles) {
-        let startDate = startDate ?? Date()
-        let endDate = endDate ?? Calendar.current.date(byAdding: .day, value: 1, to: startDate) ?? startDate
+        let startDate2 = startDate ?? Date()
```

Comment:
instead of startDate2 i think it would be better if it is calculatedStartDate ? calculatedEndDate?

Reply:
Done